Source/Reporters/JunitXMLReporter.jsl

```diff
@@ -221,15 +222,15 @@ Define Class(
       anonymous_test_cases = anonymous_test_suite << get keys;
 
       // find number of test cases in anonymous_test_suite with form label_XX where XX is any number
-      curr_anonymous_test_cases = 0;
+      /*curr_anonymous_test_cases = 0;
```

Comment:
More dead code here.

Reply:
done in de325ccbad38866829e3e311823355d232c5b9fd